### PR TITLE
[RFR] Quick and dirty mobile fixes for documentation

### PR DESCRIPTION
**Pull request by djhi**
screen

### CODE REVIEW

**Comment by fzaninotto** [comment]
Superb. I vote for a hamburger icon instead of the "menu" label.

**Comment by djhi** [comment]
Updated with icon:
screen

**Comment by fzaninotto** [comment]
beautiful

File: docs/img/menu.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (Kmaschta) [inline docs/img/menu.svg]: Are all these groups necessary ?

Comment (djhi) [reply docs/img/menu.svg]: I just downloaded an hamburger image somewhere

**Comment by fzaninotto** [comment]
To be backported to master.

**Comment by fzaninotto** [comment]
Refs #1269

**Pull request merged**